packages/docusaurus-theme-classic/src/theme/LastUpdated/index.tsx

```diff
@@ -8,23 +8,37 @@
 import React from 'react';
 import Translate from '@docusaurus/Translate';
 import {ThemeClassNames} from '@docusaurus/theme-common';
+import {useDateTimeFormat} from '@docusaurus/theme-common/internal';
 import type {Props} from '@theme/LastUpdated';
 
 function LastUpdatedAtDate({
   lastUpdatedAt,
-  formattedLastUpdatedAt,
 }: {
   lastUpdatedAt: number;
-  formattedLastUpdatedAt: string;
 }): JSX.Element {
+  const convertedLastUpdateAt = lastUpdatedAt * 1000;
```

Comment:
```suggestion
  const atDate = new Date(lastUpdatedAt * 1000);
```

Convert to date directly here, instead of doing it twice later